src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -452,6 +452,23 @@ const JupyterFrameManager = ({ onClose, frameRef }) => {
   return null
 }
 
+const PeriodicCookieSetter = ({ namespace, clusterName }) => {
+  const signal = useCancellation()
+
+  const periodicallySetCookie = async () => {
+    while (!signal.aborted) {
+      withErrorIgnoring(() => Ajax().Jupyter.notebooks(namespace, clusterName).setCookie())
```

Comment:
`withErrorIgnoring` returns a function which is currently never called.
```suggestion
      withErrorIgnoring(() => Ajax().Jupyter.notebooks(namespace, clusterName).setCookie())()
```